README.md

```diff
@@ -23,31 +27,121 @@ const job = await client.createReport({
   endTime: new Date().toISOString(),
   browserName: "Chrome",
   browserVersion: "105",
-  framework: "playwright",
+  framework: "playwright", // alternatives: cypress, playwright, testcafe
   frameworkVersion: "1.25.0",
   platformName: "Windows 11"
 });
 console.log(job.id); // the job ID
 console.log(job.url); // the full URL of the job
 ```
 
-## Upload Assets
+## Upload log for JavaScript Batch Frameworks
+
+The minimum `sauce-test-report.json` file to fullfill the web UI:
+
+```json
+{
+  "status": "passed",
+  "attachments": [],
+  "suites": [
+    {
+      "name": "somegroup",
+      "status": "passed",
+      "metadata": {},
+      "suites": [
+        {
+          "name": "somefile.test.js",
+          "status": "passed",
+          "metadata": {},
+          "suites": [],
+          "attachments": [],
+          "tests": [
+            {
+              "name": "yay",
+              "status": "passed",
+              "duration": 123,
+              "startTime": "2023-07-06T22:03:19.591Z",
+              "attachments": [],
+              "metadata": {}
+            }
+          ]
+        }
+      ],
+      "attachments": [],
+      "tests": []
+    }
+  ],
+  "metadata": {}
+}
+```
+To upload `sauce-test-report.json`:
 
 ```javascript
-const Readable = require('stream').Readable;
+const uploads = await client.uploadAssets(
+  job.id,
+  [{
+    filename: "log.json",
+    data: fs.createReadStream("log.json")
+  }]
+);
+// Upon success, the array `uploads.uploaded` will equal in length to the number of assets you intended to upload.
+// Individual assets that fail to upload will be reported via `uploads.errors`.
+```
 
-const client = new TestComposer({
-  region: 'us-west-1',
-  username: process.env.SAUCE_USERNAME,
-  accessKey: process.env.SAUCE_ACCESS_KEY,
-  headers: {'User-Agent': `your-fancy-reporter/1.2.3`}
+## Create Reports for Selenium
+
+```javascript
+const job = await client.createReport({
+  name: "My Fancy Job!",
+  passed: true,
+  startTime: new Date().toISOString(),
+  endTime: new Date().toISOString(),
+  browserName: "Chrome",
+  browserVersion: "105",
+  framework: "webdriver",
+  platformName: "Windows 11"
 });
+console.log(job.id); // the job ID
+console.log(job.url); // the full URL of the job
+```
+
+## Upload Log for Selenium job
 
-const s = new Readable();
-s.push('hello!');
-s.push(null);
+The minimum `log.json` file to fullfill the web UI:
 
-const uploads = await client.uploadAssets(job.id, [{filename: "console.log", data: s}]);
+```json
+[
+	{
+    "screenshot": null,
+    "suggestion_values": [
+
+    ],
+    "start_time": 1688673799.516,
+    "request": {
+      "using": "css selector",
+      "value": "[id=\"onetrust-accept-btn-handler\"]"
+    },
+    "result": {
+      "message": "no such element",
+      "error": "no such element"
```

Comment:
You say this is the minimal input, but can't result be empty? Wouldn't that make this optional?

Would be also good to add some description as to what these fields mean.
Like `"statusCode": 1`. What values are acceptable?